sdk/containerregistry/azure-containerregistry/azure/containerregistry/_models.py

```diff
@@ -156,11 +155,21 @@ def _from_generated(cls, generated):
             manifest_count=generated.manifest_count,
             tag_count=generated.tag_count,
             content_permissions=generated.writeable_properties,
-            registry=generated.additional_properties.get("registry", None),
+        )
+
+    def _to_generated(self):
+        # type: () -> GeneratedRepositoryProperties
+        return GeneratedRepositoryProperties(
+            name=self.name,
+            created_on=self.created_on,
+            last_updated_on=self.last_updated_on,
+            manifest_count=self.manifest_count,
+            tag_count=self.tag_count,
+            writeable_properties=self.writeable_properties._to_generated(),  # pylint: disable=protected-access
         )
 
 
-class RegistryArtifactOrderBy(str, Enum):
+class ManifestOrderBy(str, Enum):
```

Comment:
I feel like this should just be called `ManifestOrder` or `ManifestOrdering`.
Then it reads better: `order_by=ManifestOrder.LAST_UPDATE_TIME_DESCENDING`